roles/edpm_bootstrap/defaults/main.yml

```diff
@@ -72,6 +72,9 @@ edpm_bootstrap_selinux_mode: enforcing
 # Swap management
 edpm_bootstrap_swap_size_megabytes: 1024
 edpm_bootstrap_swap_path: /swap
+# swap path needs to be under /var on bootc, so a separate default value is
+# needed.
+edpm_bootstrap_swap_path_bootc: /var/swap
```

Comment:
so other then the upgrade impact of needing to move the swap file
there is no reason we could not use /var/swap for all deployments.

we may want to consider doing that.

for example  we could define new vars like this
```
edpm_bootstrap_legacy_swap_path: /swap
edpm_bootstrap_swap_path: /var/swap
```
then check if edpm_bootstrap_legacy_swap_path exists and edpm_bootstrap_swap_path does not.
if that is the case check fi there is enough space to allocate the second swap file, do swap on to the new location and then swappoff the old location and delete the legacy swap file.

this can be a followup pr and could be deferred to the next reboot also.

but that would give us a way to normalise the two flows.

Reply:
It would indeed be nice to bring consistency to both types of RHEL nodes. Just a small inconvenience during the transition

Reply:
@bshephar could you please create a jira tracker for that?

Reply:
i would suggeest 

1 create new swap file
2 swapon new file
3 swapoff old file
4 delete old file

that should alow the kernel to move the swap pages between the fiels without impacting the workloads.

has the issue with 0 downtime update been fixed in bootc yet?
if not then any updates to the bootc image woudl need you to drain the compute node first
in which case the swap file should not be actively used so you should be able to just nuke the old one and create the new one in image mode as @bshephar suggets.